wurst/util/UnitUtils.wurst

```diff
@@ -0,0 +1,60 @@
+package UnitUtils
+import BuffObjEditing
+import ClosureTimers
+
+
+
+/*	This file contains various utility functions for Unit, 
+	which require enhanced functionality like closures and cannot be defined in Unit.wurst */
```

Comment:
Is this something @Frotty  has suggested?

Reply:
I was told anything using closures should go into separate XUtils.wurst file, never to the handles file itself if that's what you're asking.

Reply:
Sounds good to me